.mkosi/mkosi.opensuse

```diff
@@ -63,3 +63,14 @@ BuildPackages=
 Packages=
         # brought in via meson->python3
         libp11-kit0
+        # --bootable=no
+        dbus-1
+        libapparmor1
+        libcrypt1
+        libcryptsetup12
+        libkmod2
+        liblz4-1
+        libmount1
+        libseccomp2
```

Comment:
hmm, all these libraries must be deps of systemd.rpm, no? ence there is no need to list them here explicitly, is there? can you explain?

Reply:
The base image pattern is really stripped down, so when `--bootable=no`, there's no dracut and no systemd installed. I can predict explicit listing may turn into maintenance burden, OTOH, this way it ensures that only systemd files from the build proper are installed and tested. (This of course doesn't hold for `--bootable=yes`, and e.g. distro pam_systemd.so is used instead of the built one because of specific paths configuration.)

Reply:
hmm, no systemd at all?

on fedora you do get libsystemd.so at the very least which pulls many of these in.

Reply:
It's a base pattern for application containers, hence such basic deps. It results in ~200 MiB image. Do you suggest using more expanded basic pattern?